shared/src/bundles/user-details/validation-schemas/user-details-approve.validation-schema.ts

```diff
@@ -0,0 +1,11 @@
+import joi from 'joi';
+
+import { type UserDetailsApproveRequestDto } from '../types/types.js';
+
+const userDetailsApprove = joi.object<UserDetailsApproveRequestDto, true>({
+    id: joi.string().trim().required(),
+    isApproved: joi.boolean().required(),
+    deniedReason: joi.string(),
+});
+
+export { userDetailsApprove };
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !